starter-code/css/main.css

```diff
@@ -6,3 +6,206 @@ Green: #00B172
 White: #FFF
 
 */
+
+@import url('https://fonts.googleapis.com/css?family=Quicksand');
+
+html, body, div, span, applet, object, iframe,
```

Comment:
Intentad coger el reset del cheatsheet, ahora mismo no necesitáis tooooda estas cosas, añade solamente líneas de código más.